### PR TITLE
WIP : Replace route with intercept

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
replacing route with intercept
Some tests modified to handle this change.

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

**Comment by hetunandu** [comment]
@NandanAnantharamu Please resolve the conflicts and run the test suite again

File: app/client/cypress/support/commands.js [hunk 1]
```diff
@@ -23,7 +23,11 @@ const datasource = require("../locators/DatasourcesEditor.json");
 
 let pageidcopy = " ";
 
-Cypress.Commands.add("createOrg", () => {
+function checkIfApiWasSuccessful(interception, statuscode) {
```
Comment (hetunandu) [inline app/client/cypress/support/commands.js:26]: Rename to `assertApiResponseStatusCode`

**Review by hetunandu — CHANGES_REQUESTED** [review]
Minor change

**Review by arslanhaiderbuttar — APPROVED** [review]

**Review by mohanarpit — DISMISSED** [review]

**Comment by mohanarpit** [comment]
Approving to run the tests

**Review by yatinappsmith — DISMISSED** [review]

**Review by yatinappsmith — APPROVED** [review]
LGTM

**Comment by NandanAnantharamu** [comment]
closing this as its not relevant

**Pull request closed without merging**